packages/babel-preset-react-app/index.js

```diff
@@ -84,10 +84,13 @@ if (env === 'test') {
   module.exports = {
     presets: [
       // Latest stable ECMAScript features
-      [require.resolve('babel-preset-latest'), {
-        'es2015': {
-          modules: false
-        }
+      [require.resolve('babel-preset-env'), {
+        targets: {
+          // We support everything React supports
+          'ie': 9
+        },
+        // Disable polyfill transforms
+        useBuiltIns: false
```

Comment:
Enabling it will transform `import babel-polyfill` into only the necessary polyfills/transforms needed, based on targets you specify.